src/app/people/people.component.html

```diff
@@ -45,72 +41,79 @@ <h3>You may bring an unlimited number of mentors.</h3>
         </div>
-        <div class="half unit row">
-            <fieldset>
-                <input type="button" value="Add Mentor." (click)="addMentor()" [disabled]="!mentorDetailsCompleted">
-            </fieldset>
+        <div class="two unit row">
+            <div class="one half unit column">
+                <button (click)="addMentor()" [disabled]="!mentorDetailsCompleted">Add Mentor.</button>
+            </div>
         </div>
-        <div class="half unit row" *ngFor="let mentor of mentors; let i = index">
-            <fieldset>
+        <fieldset class="one unit persistent row content" *ngFor="let mentor of mentors; let i = index">
+            <div class="two unit column">
                 <input type="text" [(ngModel)]="mentors[i].name" (ngModelChange)="onMentorChanges()" placeholder="Name">
+            </div>
+            <div class="three unit column">
                 <input type="text" [(ngModel)]="mentors[i].email" (ngModelChange)="onMentorChanges()" placeholder="Email">
+            </div>
+            <div class="one half unit column">
                 <input type="text" [(ngModel)]="mentors[i].phone" (ngModelChange)="onMentorChanges()" placeholder="Phone">
-                <input type="button" value="Delete." (click)="deleteMentor(i)">
-            </fieldset>
-        </div>
-    </div>
-</section>
-
-<section *ngIf="recruiterLimit > 0">
-    <div class="container width">
-        <div class="one unit row">
-            <h2>Recruiters</h2>
+            </div>
+            <div class="one unit column">
+                <button (click)="deleteMentor(i)">Delete!</button>
+            </div>
+        </fieldset>
+        <div class="two unit row" *ngIf="recruiterLimit > 0">
+            <div class="unit column grows">
+                <h4>Recruiters</h4>
+            </div>
         </div>
-        <div class="one unit row">
-            <h3>You may bring {{ recruiterLimit }} recruiters.</h3>
+        <div class="two unit row" *ngIf="recruiterLimit > 0">
+            <div class="unit column grows">
+                <p>You may bring {{ recruiterLimit }} recruiters.</p>
+            </div>
         </div>
-        <div class="half unit row">
-            <fieldset>
-                <input type="button" value="Add Recruiter." (click)="addRecruiter()" [hidden]="recruiters ? recruiters.length >= recruiterLimit : true" [disabled]="!recruiterDetailsCompleted">
-            </fieldset>
+        <div class="two unit row" *ngIf="recruiterLimit > 0">
+            <div class="one half unit column">
+                <button (click)="addRecruiter()" [hidden]="recruiters ? recruiters.length >= recruiterLimit : true" [disabled]="!recruiterDetailsCompleted">Add Recruiter.</button>
+            </div>
         </div>
-        <div class="half unit row" *ngFor="let recruiter of recruiters; let i = index">
-            <fieldset>
+        <fieldset class="one unit persistent row content" *ngFor="let recruiter of recruiters; let i = index">
```

Comment:
Again, the list of recruiters gets quite cramped, can we have a margin between each recruiter?